### PR TITLE
Adds ssm text to header

### PR DESCRIPTION
**Pull request by mseidl**
## What does this PR change?
Some system list pages have an automagic adding/removing from the SSM by selecting/deselecting the checkbox.  This pr adds the "SSM" text to the table header to make it clear that it's an SSM checkbox. 

## GUI diff

No difference.

Before:
image

After:
image

No header text on pages with the add to ssm button

image

- [ ] **DONE**

## Documentation
- No documentation needed: I will ping @Loquacity about updating screen shots. 

- [x] **DONE**

## Test coverage
- No tests: not needed 

- [ ] **DONE**

## Links

Fixes #
Tracks # **add downstream PR, if any**

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle" 
- [ ] Re-run test "java_pgsql_tests" 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: java/spacewalk-java.changes [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Add ssm text to checkbox table header on system list pages that don't have add to ssm button
```
Comment (ncounter) [inline java/spacewalk-java.changes:1]: I'd better rephrase this to be clear to users.
Instead of
`.Add ssm text to checkbox table header on system list pages that don't have add to ssm button`
replace it with
`Clarify the behavior of the checkbox system list, when it adds systems to ssm`

Comment (mseidl) [reply java/spacewalk-java.changes:1]: ok. i'll do that. thanks.
